Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -595,6 +595,76 @@ Messages::WebDriverClient::NewWindowResponse WebDriverConnection::new_window(Jso
     return result;
 }
 
+// 11.6 Switch To Frame, https://w3c.github.io/webdriver/#dfn-switch-to-frame
+Messages::WebDriverClient::SwitchToFrameResponse WebDriverConnection::switch_to_frame()
+{
+    // FIXME: 1. Let id be the result of getting the property "id" from parameters.
+
+    // FIXME: 2. If id is not null, a Number object, or an Object that represents a web element, return error with error code invalid argument.
+
+    // FIXME: 3. Run the substeps of the first matching condition:
+
+    // -> id is null
+
+        // FIXME: 1. If session's current top-level browsing context is no longer open, return error with error code no such window.
+
+        // FIXME: 2. Try to handle any user prompts with session.
+
+        // FIXME: 3. Set the current browsing context with session and session's current top-level browsing context.
+
+    // -> id is a Number object
```

Comment:
The linter really doesn't like the indented comments. You have a couple of options, but one simple one is to wrap the sub-steps in `{}` so that indentation happens automatically. An empty `{}` in C++ does nothing so it's harmless.